app/grandchallenge/algorithms/forms.py

```diff
@@ -111,6 +111,7 @@ class Meta:
             "social_image",
             "public",
             "use_flexible_inputs",
+            "repo_name",
```

Comment:
Just to double check - do we want to make this user editable?

Reply:
I think that's the way to go. It's not foolproof, but it allows users to change the repo if they have already setup the installation and it is part of solving the issue if an admin user needs to approve the installation before linking it on gc.